nncf/torch/engine.py

```diff
@@ -0,0 +1,38 @@
+"""
+ Copyright (c) 2023 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import torch
+from typing import Union, Any, Dict
+from torch import nn
+
+from nncf.common.engine import Engine
+
+
+class PTEngine(Engine):
+    """
+    Engine for Torch backend.
+    """
```

Comment:
Please, add the parameters to the docstring.

Reply:
Done, please check